BREEZE.rst

```diff
@@ -290,104 +275,85 @@ It is as easy as copy&pasting this line into your code:
-to the webserver (you can use ``tmux`` to multiply terminals).
+Start the webserver manually with the ``airflow webserver`` command if you want to connect
+to the webserver. You can use ``tmux`` to multiply terminals.
 
-For databases you need to run ``airflow db reset`` at least once (or run some tests) after you started
-Airflow Breeze to get the database/tables created. You can connect to databases
-with IDE or any other Database client:
+For databases, you need to run ``airflow db reset`` at least once (or run some tests) after you started
+Airflow Breeze to get the database/tables created. You can connect to databases with IDE or any other database client:
 
 .. image:: images/database_view.png
     :align: center
     :alt: Database view
 
-You can change host port numbers used by setting appropriate environment variables:
+You can change the used host port numbers by setting appropriate environment variables:
 
 * ``WEBSERVER_HOST_PORT``
 * ``POSTGRES_HOST_PORT``
 * ``MYSQL_HOST_PORT``
 
-When you set those variables, next time when you enter the environment the new ports should be in effect.
+If you set these variables, next time when you enter the environment the new ports should be in effect.
+
+Cleaning Up the Environment
+---------------------------
+
+You may need to clean up your Docker environment occasionally. The images are quite big
+(1.5GB for both images needed for static code analysis and CI tests) and, if you often rebuild/update
+them, you may end up with some unused image data.
+
+To clean up the Docker environment:
 
-Cleaning up the images
-----------------------
+1. `Stop Breeze <#stopping-breeze>`_ with ``./breeze --stop-environment``.
 
-You might need to cleanup your Docker environment occasionally. The images are quite big
-(1.5GB for both images needed for static code analysis and CI tests). And if you often rebuild/update
-images you might end up with some unused image data.
+2. Run the ``docker system prune`` command.
 
-Cleanup can be performed with ``docker system prune`` command.
-Make sure to `Stop Breeze <#stopping-breeze>`_ first with ``./breeze --stop-environment``.
+3. Run ``docker images --all`` and ``docker ps --all`` to verify that your Docker is clean.
 
-If you run into disk space errors, we recommend you prune your docker images using the
-``docker system prune --all`` command. You might need to restart the docker
-engine before running this command.
+   Both commands should return an empty list of images and containers respectively.
 
-You can check if your docker is clean by running ``docker images --all`` and ``docker ps --all`` - both
-should return an empty list of images and containers respectively.
+If you run into disk space errors, consider pruning your Docker images with the ``docker system prune --all`` command. You may need to restart the Docker Engine before running this command.
```

Comment:
is it different from step 2? we don't recommend --all for step 2, do we?

Reply:
usually when you run out of disk space in Docker I'd say it's safer to prune all - or even reinstall docker. It might get into some weird state if that happens.